src/runner/run-primary-async.ts

```diff
@@ -201,7 +201,9 @@ export async function runPrimaryAsync(options: ConvertCommandCliArgs) {
           (flowFilePath.fileType === FlowFileType.NO_FLOW &&
             options.skipNoFlow) ||
           (flowFilePath.fileType === FlowFileType.NO_ANNOTATION &&
-            !options.convertUnannotated);
+            !options.convertUnannotated) ||
+          // skip .js(x) files that have a corresponding .d.ts file
+          fs.existsSync(flowFilePath.filePath.replace(/\.jsx?$/, ".d.ts"));
```

Comment:
The code I had in the first commit looked like it should've worked, but I suspect that a copy of the object I was mutating was being made some where that prevent that value from showing up here.  This is a little more expensive since we do the same `fs.existSync` call twice, but it didn't seem to affect the running time in any significant way.